modules/rtm/R/invert.custom.R

```diff
@@ -142,6 +151,11 @@ invert.custom <- function(observed, invert.options,
                       lower = param.mins, upper = param.maxs)
     inits <- fit$par
   }
+  if (!all(diag(init.Jump) > 0)) {
+    warning("Passed init.Jump matrix with zero values on diagonals. ", 
```

Comment:
I kept these separate to isolate problems between Jump distributions that are passed as inputs, which can be illegal (they just get dropped altogether), with the default initial Jump distributions that are set by the code, which _has_ to be legal and is therefore an error.